internal/indexcoord/node_manager.go

```diff
@@ -102,34 +105,35 @@ func (nm *NodeManager) AddNode(nodeID UniqueID, address string) error {
 }
 
 // PeekClient peeks the client with the least load.
-func (nm *NodeManager) PeekClient(meta Meta) (UniqueID, types.IndexNode) {
-	log.Debug("IndexCoord NodeManager PeekClient")
-
+func (nm *NodeManager) PeekClient(meta *Meta) (UniqueID, types.IndexNode) {
+	// just for log, not used.
 	dataSize, err := estimateIndexSizeByReq(meta.indexMeta.Req)
```

Comment:
suggset to delete this. if estimateIndexSizeByReq failed, it affect the peek opeartion.

Reply:
ok